### PR TITLE
Add support for snapcraft

### PR DESCRIPTION
**Pull request by theOriginalMuktaka**
I added support for snapcraft builds following the instructions [here](https://gist.github.com/popey/7803cfbda9b337fab792c403e50e4709).
Please tell me whether I should add a snapcraft build to `.circleci/config.yml`.
I can install and run the built snap successfully on Ubuntu 16.04 LTS.

I'm new to yarn so I'm trusting the changes that yarn made to the yarn.lock file.

### CODE REVIEW

File: packages/graphql-playground-electron/package.json [hunk 1]
```diff
@@ -130,7 +131,7 @@
     "concurrently": "3.5.1",
     "css-loader": "0.23.1",
     "electron": "1.8.1",
-    "electron-builder": "19.49.2",
+    "electron-builder": "^19.53.0",
```
Comment (timsuchanek) [inline packages/graphql-playground-electron/package.json]: Please rebase, the dependencies have already been updated

**Comment by theOriginalMuktaka** [comment]
I currently have a very limited understanding of docker (I'm a high-schooler who participated in Google Code-in and this problem made me read up on docker for the first time). I also do not have a Mac. So I'm finding it hard to get to the root of this problem.

However, I understand that the "deb" target builds without problems in the same setup. Are you using a virtualized Linux distribution on your computer?

In that case, please try the equivalent of "sudo install snapcraft" and try again. Please tell me whether it builds after doing this.

**Comment by timsuchanek** [comment]
Thanks for the PR, but the dependencies got outdated by now. I'm adding this change in a separate commit, from the next version on we will have snapcraft support 🙂

**Comment by gustawdaniel** [comment]
@timsuchanek can you add to documentation how to install by snapcraft? Now only brew is described.

**Pull request closed without merging**